apps/console/src/features/server-configurations/components/governance-connectors/dynamic-governance-connector.tsx

```diff
@@ -151,6 +151,20 @@ export const DynamicGovernanceConnector: FunctionComponent<DynamicGovernanceConn
         />
     );
 
+    /**
+     * Default illustration for connector illustrations.
+     */
+    const resolveConnectorIllustration = (id: string): ReactElement => {
+
+        const illustration = get(GovernanceConnectorIllustrations, id);
+
+        if (!illustration) {
+            return GovernanceConnectorIllustrations.default;
+        }
+
+        return illustration;
+    }
```

Comment:
Unterminated statement.
```suggestion
    };
```

Reply:
Fixed with efe4b13